### ISSUE TITLE
Emacs spaces are stuck

### ISSUE THREAD

**Issue by sandric** [microsoft/terminal#1474]
# Environment

```none
Windows build number: 10.0.18922.1000
Windows Terminal version (if applicable): 0.2.1715.0

Any other software? Emacs
```

# Steps to reproduce

In Ubuntu wsl 2 session open emacs. Try to enter some characters in new file and then press space. Try to press is couple of times, press and hold.

# Expected behavior

Space characters entered as expected.

# Actual behavior

Spaces not entered and cursor stuck, after second or two - spaces are entered and cursor jumps to normal. Can not reproduce in simple bash/fish commandline, for now seen only in emacs.

**Comment by sandric** [comment]
Ok, quick searching and I found that: https://github.com/microsoft/Terminal/issues/283. By changing TERM to "rxvt" it looks like it fixes stuck spaces issue, but instead I now dont have HOME/END/SHIFT-ARROW etc. for some reason. Although I can make workaround for this via redefining basic keystrokes, but I would like to know is its knows issue, should there be fix, other TERM settings?

**Comment by sandric** [comment]
Ok, another update: it flawlessly works in tmux. Since I'm prefer to work with it - for me issue is closed for now I guess (dont know for future, just start playing with new terminal). So I dont know, should it be closed or should it stay for reminder porpuses.

P.S. I haven't tested but I guess it will work also with screen multiplexer. Cheers.

**Comment by zadjii-msft** [comment]
Since we don't have an active repro for this bug anymore, and it's possible that it was fixed by some combo of #283/#1825, I'm going to close this. If it does re-appear, we can always reopen to investigate.
